components/FooterContainer/FooterContainer.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
```

Comment:
is it worth writing a test for this?

Reply:
Could do but not sure if we'd get a lot from it? It's just a wrapper for <Footer/ > which is already tested.

Reply:
It's certainly not doing much, but I wonder if we should write a snapshot test in order to make sure that the prop we're passing it doesn't change and cause regressions? More like an integration test to ensure we're plugging everything into it correctly.